shared/common-adapters/markdown/shared.tsx

```diff
@@ -145,11 +145,12 @@ const rules: {[type: string]: SimpleMarkdown.ParserRule} = {
   },
   fence: {
     // aka the ``` code blocks
+    // must agree with ReplaceQuotedSubstrings in go/chat/utils/utils.go
     ...SimpleMarkdown.defaultRules.fence,
     match: SimpleMarkdown.anyScopeRegex(/^```(?:\n)?((?:\\[\s\S]|[^\\])+?)```(?!`)(\n)?/),
     // original:
     // match: SimpleMarkdown.blockRegex(/^ *(`{3,}|~{3,}) *(\S+)? *\n([\s\S]+?)\s*\1 *(?:\n *)+\n/),
-    // ours: three ticks (anywhere) and remove any newlines in front and one in back
+    // ours: three ticks (anywhere) and ignore up to one line feed at the start and at the end
```

Comment:
Have you considered replacing these Keybase-specific customizations with [CommonMark](https://spec.commonmark.org/current/)-compliant syntax?